packages/flutter/lib/src/foundation/_features.dart

```diff
@@ -23,6 +26,6 @@ bool isWindowingEnabled = debugEnabledFeatureFlags.contains('windowing');
 ///
 /// Do not use this API. Flutter can and will make breaking changes to this API.
 @internal
-Set<String> debugEnabledFeatureFlags = <String>{
+final Set<String> debugEnabledFeatureFlags = <String>{
```

Comment:
I feel like the lack of `final` should have been caught by the analyzer/linter before... Good catch though.